source/agora/node/FullNode.d

```diff
@@ -692,14 +692,15 @@ public class FullNode : API
         immutable Genesis = () {
             if (!config.node.testing)
                 return COINNET.GenesisBlock;
-            if (!config.node.limit_test_validators)
+            if (!config.node.test_validators
+                || config.node.test_validators == TESTNET.GenesisBlock.header.enrollments.length)
```

Comment:
Why?

Reply:
It's my suggestion. I don't see the meaning of `limit` so I suggested it. Doesn't the meaning of the variable `limit_test_validators` change in this PR?

Reply:
It is still limit of the available `validators` included in the `GenesisBlock` but I thought it was also a bit too long and still makes sense.

Reply:
^^ The extra check in the condition is that if all the `validators` are being used there is no need to build a different `GenesisBlock`. Not really needed though.

Reply:
Removed extra check as the created Genesis will be the same anyway.